hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockInfo.java

```diff
@@ -136,7 +136,7 @@ BlockInfo getPrevious(int index) {
     BlockInfo info = (BlockInfo)triplets[index*3+1];
     assert info == null ||
         info.getClass().getName().startsWith(BlockInfo.class.getName()) :
-        "BlockInfo is expected at " + index*3;
+        "BlockInfo is expected at " + (index*3+1);
```

Comment:
nit: Better to have some space around the values
```suggestion
        "BlockInfo is expected at " + (index * 3 + 1);
```

Reply:
Thanks @ayushtkn for your comments.